index.html

```diff
@@ -11,6 +11,156 @@
     <title>WineTime</title>
 </head>
 <body>
+    <header>
+        <section class="header-wrapper--orange">
+            <div class="header-top content-width">
+                <div class="header-top-left">
+                    <i class="fa fa-phone" aria-hidden="true"></i><a class="header-top-text" href="tel:+1800229933">044 494 0324</a>
```

Comment:
why number in attribute is different from text?
and you could do just <input type="tel" readonly>
but it's as you wish, just information